From 74544b4ead6dfc8645b3f0bce6787a494f254cc5 Mon Sep 17 00:00:00 2001
From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Sun, 22 Nov 2015 18:37:58 +0100
Subject: [PATCH] networkd: link - fix reading routes

This regressed during the port to extract_first_word in c598ac76
(v228).

CID #1338083
---
 src/network/networkd-link.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/network/networkd-link.c b/src/network/networkd-link.c
index 4af895a..64a4b74 100644
--- a/src/network/networkd-link.c
+++ b/src/network/networkd-link.c
@@ -2327,6 +2327,8 @@ network_file_fail:
         }
 
         if (routes) {
+                p = routes;
+
                 for (;;) {
                         Route *route;
                         _cleanup_free_ char *route_str = NULL;
-- 
1.9.1

